Makefile

```diff
@@ -63,8 +63,21 @@ test-dlk-main: build
 	# Run dlk test
 	docker run --rm -t -v $(HOME)/.ssh:/tmp/.ssh -e FPGA_HOST --net=host $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "cp -R /tmp/.ssh /root/.ssh && apt-get update && apt-get install -y iputils-ping && cd dlk && python setup.py test"
 
+.PHONY: os-docker
```

Comment:
`os-docker` is not good for me.
How about `rootfs-docker` and other commands following `rootfs-xxx`?.